src/us/kbase/workspace/test/workspace/WorkspaceTest.java

```diff
@@ -3568,23 +3554,23 @@ public void saveLargeProvenance() throws Exception {
 			methparams.add(TEXT1000);
 		}
 		Provenance p = new Provenance(foo);
-		p.addAction(new ProvenanceAction().withMethodParameters(methparams));
+		p.addAction(ProvenanceAction.getBuilder().withMethodParameters(methparams).build());
 		ws.saveObjects(foo, prov, Arrays.asList( //should work
 				new WorkspaceSaveObject(getRandomName(), data, SAFE_TYPE1, null, p, false)),
 				getIdFactory());
 		
 		
 		methparams.add(TEXT1000);
 		Provenance p2 = new Provenance(foo);
-		p2.addAction(new ProvenanceAction().withMethodParameters(methparams));
+		p2.addAction(ProvenanceAction.getBuilder().withMethodParameters(methparams).build());
 		try {
 			ws.saveObjects(foo, prov, Arrays.asList(
-					new WorkspaceSaveObject(getRandomName(), data, SAFE_TYPE1, null, p, false)),
+					new WorkspaceSaveObject(getRandomName(), data, SAFE_TYPE1, null, p2, false)),
 					getIdFactory());
 			fail("saved too big prov");
 		} catch (IllegalArgumentException iae) {
 			assertThat("correct exception", iae.getLocalizedMessage(),
-					is(String.format("Object #1, %s provenance size 1000318 exceeds limit of " +
+					is(String.format("Object #1, %s provenance size 1000322 exceeds limit of " +
```

Comment:
Switched some `[]` to `null` in the json string that's used to calculate the size